src/Model/ResourceModel/Shopgate/Order.php

```diff
@@ -70,4 +70,28 @@ public function getByOrderId($id)
 
         return $connection->fetchRow($select, $bind);
     }
+
+    /**
+     * Filters for all orders that are not already synchronized to Shopgate
+     *
+     * @return Order
+     */
+    public function getUnsynchronizedOrders()
```

Comment:
naming convention is off, name assumes that the orders will be retrieved. Maybe something like "filterByUnsynchronizedOrders()`, same with the other one